notebooks/official/tensorboard/vertex_tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,856 @@
+{
```

Comment:
<p>Use the standardized way to set a service account otherwise the service account won't be set up correctly in cloud build tests</p><pre class="ql-syntax" spellcheck="false">SERVICE_ACCOUNT = "[your-service-account]"  # @param {type:"string"}
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>

Reply:
<p>To confirm will cloud build tests inject certain service account name  for "[your-service-account]", or we have to do something like</p><p>if SERVICE_ACCOUNT == "[your-service-account]"</p><p>    SERVICE_ACCOUNT = account generated from timestamp</p>